### DISCUSSION TITLE
Access to billing address

### DISCUSSION THREAD

**Discussion by patjakubik** [Shopify/function-examples#367] [API Ideas]
We would like to perform extra validation on the billing address, but it is not a part of the schema. It is accessible through checkout extensions, so I thought it would make sense if we could have it in functions too.

**Comment by shreyankgw** [comment]
Completely agree on this. The thing with checkout extensions though is they are limited to shopify checkout only, if someone uses google pay/apple pay, this validation rule will still be absent. We are currently going through this, hope shopify soon will have the support for this in shopify functions.

**Comment by bithaolee** [comment]
Some customers sell digital products and don't have shipping address, but need to validate the billing address field, which is currently not possible due to the inability to read the billing address.

**Comment by bithaolee** [comment]
There are already dozens of customers who need to validate the billing address. I don't understand why the Validation function can't be permitted to read the billing address.

**Comment by evaldas-raisutis** [comment]
Any update of this?
